spring-cloud-aws-cloudmap/src/main/java/org/springframework/cloud/aws/cloudmap/CloudMapDiscoveryProperties.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.cloudmap;
+
+import java.util.Map;
+
+/**
+ * POJO class to capture cloudmap discovery attributes.
+ *
+ * @author Hari Ohm Prasath
+ * @since 1.0
+ */
+public class CloudMapDiscoveryProperties {
+
+	private String serviceNameSpace;
```

Comment:
`NameSpace` is usually just a single word

Reply:
I have updated all the `serviceNameSpace` names to `nameSpace` to keep it consistent